sipa/utils/__init__.py

```diff
@@ -82,6 +88,38 @@ def support_hotline_available():
         return False
 
 
+@cached(cache=TTLCache(maxsize=1, ttl=300))
+def meetingcal():
+    url = "https://agdsn.de/cloud/remote.php/dav/public-calendars/bgiQmBstmfzRdMeH?export"
+
+    try:
+        response = requests.get(url, timeout=1)
+    except requests.exceptions.RequestException:
+        return []
+        
+    if response.status_code != 200:
+        return []
+    
+    try:
+        calendar = icalendar.Calendar.from_ical(response.text)
+    except ValueError:
+        return []
+
+    events = recurring_ical_events.of(calendar).between(datetime.now(), datetime.now() + relativedelta(months=1))
+
+    next_meetings = []
+    for event in events:
+        next_meetings.append({
+            "title": event["SUMMARY"],
+            "datetime": event["DTSTART"].dt,
+            "location": event["LOCATION"] if "LOCATION" in event else "-",
+            "location_link": markdown.markdown(event["LOCATION"]) if "LOCATION" in event else "-"
+        })
+
+    next_meetings = sorted(next_meetings, key=lambda x: x["datetime"])
```

Comment:
Here you can use an itemgetter (`from operator import itemgetter`):
```suggestion
    next_meetings = sorted(next_meetings, key=itemgetter("datetime"))
```

Reply:
resolved in 0cef0ed